smtp_connector.py

```diff
@@ -232,6 +232,20 @@ def _validate_integer(self, action_result, parameter, key, allow_zero=False):
 
         return phantom.APP_SUCCESS, parameter
 
+    def _validate_sender_email(self, action_result, input_data):
+        # sender emails also have additional restriction
+        # to not include splunk related terms in the domain name
+        restricted_domains = ["splunk", "cisco", "phantom"]
+        domain = input_data.split("@")[-1].lower()
+
+        if any(restricted_domain in domain for restricted_domain in restricted_domains):
+            return action_result.set_status(
+                phantom.APP_ERROR,
+                "The domain provided in email is restricted, please use a different email in 'from' field."
```

Comment:
```suggestion
                f"The domain provided in email is restricted (contains one of: {restricted_domains}). Please use a different email in the 'from' field."
```

Reply:
Updated the error message in the code as suggested.